### PR TITLE
Adding grommet & code highlighter components

### PR DESCRIPTION
**Pull request by mirshko**

### CODE REVIEW

File: gatsby/package.json [hunk 1]
```diff
@@ -19,11 +19,16 @@
     "gatsby-plugin-sitemap": "^2.0.5",
     "gatsby-source-filesystem": "^2.0.20",
     "gatsby-transformer-sharp": "^2.1.13",
+    "grommet": "^2.4.0",
+    "grommet-icons": "^4.2.0",
```
Comment (rdinicut) [inline gatsby/package.json]: do we need grommet-icons?

**Comment by rdinicut** [comment]
LGTM. if do do use travis anymore we can remove it IMO.

File: gatsby/package.json (hunk 1 above)
Comment (mirshko) [reply gatsby/package.json]: good point not needed atm

Comment (mirshko) [reply gatsby/package.json]: removed for now

**Pull request merged**